app/views/tasks/_task_summary.html.erb

```diff
@@ -0,0 +1,15 @@
+<% button_text ||= "The button" %>
+<% form_class ||= "" %>
```

Comment:
It looks like we're not actually using the `button_text` and `form_class` variables in the rest of this file. One example of how we could use those variables is:
```erb
<%= form_for @task, html: { class: form_class } do |f| %>
  <!-- the various form labels and inputs here -->
  <%= f.submit button_text %>
<% end %>
```